src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  color: $card-main-color;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $cart-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  transition: $hover-transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 0;
+    margin-top: 40px;
+    font-size: 12px;
+    font-style: normal;
```

Comment:
`font-style: normal;` it works by default, you can remove it

check in other places too